src/Tooltip/__tests__/Tooltip.test.js

```diff
@@ -1,141 +1,124 @@
+    const { queryByTestId, getByTestId, getByText } = render(
       <Tooltip
-        active={activeValue}
         title="Ventes nettes (ventes brutes moins les réductions et les annulations) plus les taxes
         sur la période séléctionnée."
       >
-        <button type="button">Show Tooltip</button>
+        <button type="button">{buttonText}</button>
       </Tooltip>,
     );
-    render.setProps({ active: true });
-    expect(render.state().active).toEqual(true);
-  });
+    const buttonpNode = getByText(buttonText);
 
-  it('should call click handler', () => {
-    const render = shallowWithTheme(
-      <Tooltip
-        title="Ventes nettes (ventes brutes moins les réductions et les annulations) plus les taxes
-        sur la période séléctionnée."
-      >
-        <button type="button" id="btn">
-          Show Tooltip
-        </button>
-      </Tooltip>,
-    );
-    render.find('#btn').simulate('click');
-    expect(render.state().active).toEqual(true);
-  });
+    fireEvent.click(buttonpNode);
 
-  it('should toggle', () => {
-    const render = mountWithTheme(
-      <Tooltip
-        title="Ventes nettes (ventes brutes moins les réductions et les annulations) plus les taxes
-        sur la période séléctionnée."
-      >
-        <button type="button" id="btn">
-          Show Tooltip
-        </button>
-      </Tooltip>,
-    );
-    render.find('#btn').simulate('click');
-    expect(render).toMatchSnapshot();
-    render.find('#btn').simulate('click');
-    expect(render).toMatchSnapshot();
+    let tooltipNode;
+    tooltipNode = getByTestId('tooltip');
+
+    expect(tooltipNode).toBeInTheDocument();
+
+    fireEvent.click(buttonpNode);
+
+    setTimeout(() => {
+      tooltipNode = queryByTestId('tooltip');
+      expect(tooltipNode).toBeNull();
+      done();
+    }, 500);
   });
 
-  it('should render a tooltip  displayed above the element', () => {
-    const render = mountWithTheme(
+  test('should toggle on mouseEnter/mouseLeave ', done => {
+    const buttonText = 'Show Tooltip';
+    const { queryByTestId, getByTestId, getByText } = render(
       <Tooltip
-        top
+        hover
         title="Ventes nettes (ventes brutes moins les réductions et les annulations) plus les taxes
         sur la période séléctionnée."
       >
-        <button type="button" id="btn">
-          Show Tooltip
-        </button>
+        <button type="button">{buttonText}</button>
       </Tooltip>,
     );
+    const buttonpNode = getByText(buttonText);
+
+    fireEvent.mouseEnter(buttonpNode);
+
+    let tooltipNode;
+    tooltipNode = getByTestId('tooltip');
 
-    render.find('#btn').simulate('click');
+    expect(tooltipNode).toBeInTheDocument();
 
-    expect(render).toMatchSnapshot();
+    fireEvent.mouseLeave(buttonpNode);
+
+    setTimeout(() => {
+      tooltipNode = queryByTestId('tooltip');
+      expect(tooltipNode).toBeNull();
+      done();
+    }, 500);
   });
 
-  it('should call hover handler', () => {
```

Comment:
@sun-tea I remove this test because `react-testing-library` intentionally does not give you access to the component instance. It is intended to test the app as a user would experience it